libs/gi/db/src/Database/DataManagers/OptConfigDataManager.ts

```diff
@@ -278,7 +278,7 @@ const initialBuildSettings: OptConfig = deepFreeze({
   optimizationTarget: undefined,
   mainStatAssumptionLevel: 0,
   excludedLocations: [],
-  allowLocationsState: 'unequippedOnly',
+  allowLocationsState: 'customList',
```

Comment:
I don't agree with this change, and I think we need to have a discussion here. I'd recommend reverting this change since its not material to the UI changes at hand.